### DISCUSSION TITLE
Recommended dev stack and Qt Creator setup

### DISCUSSION THREAD

**Discussion by MattiaMontanari** [fougue/mayo#299] [Q&A] [answered]
I'd like to learn a bit of Qt and Mayo seems a great playground. As a general goal I'd like to add the ability to load and render finite element meshes in Mayo, but I appreciate this may be beyond the scope of Mayo itself - and possibly timely to implement. Realistically, for now I'd be happy to debug and play around with new widgets. What software stack would you recommend?

I'm on MacOS, I have just installed Qt Creator for the first time. Do you use Qt for developing the C++ part as well, or you rely on MSVS, xcode, VS Code or others?

**Comment by HuguesDelorme** [comment, marked as answer]
Hello @MattiaMontanari
Yes I'm also using QtCreator to develop Mayo
It's a very handy and powerful IDE
